src/Plutarch/Extra/Compile.hs

```diff
@@ -3,8 +3,13 @@
 module Plutarch.Extra.Compile (mustCompile) where
 
 import qualified Data.Text as T
-import Plutarch (ClosedTerm, Config (..), TracingMode (DetTracing), compile)
-import PlutusLedgerApi.V1 (Script)
+import Plutarch (
+    Config (..),
```

Comment:
Wildcard import.

Reply:
This will actually be deleted once the Prelude PR is merged.